### PR TITLE
feat: Add doc for binary vector

### PR DESCRIPTION
**Pull request by gaocegege**
Ref https://github.com/tensorchord/pgvecto.rs-docs/issues/47

Preview: https://pgvecto-rs-docs-git-fork-gaocegege-bv-tensorchord.vercel.app/usage/vector-types.html

### CODE REVIEW

**Comment by usamoi** [comment]
There is already an unmerged PR (#45) for it.

**Comment by gaocegege** [comment]
I think we should have a separate doc for other data types, WDYT?

Of course I should drop the change to schema.md.

File: src/usage/vector-type.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+);
+
+INSERT INTO items (embedding) VALUES ('[1,0,1]'), ('[0,1,0]');
+```
+
+We support three operators to calculate the distance between two `bvector` values.
+
+- `<->` (`bvector_l2_ops`): squared Euclidean distance, defined as $\Sigma (x_i - y_i) ^ 2$.
+- `<#>` (`bvector_dot_ops`): negative dot product, defined as $- \Sigma x_iy_i$.
+- `<=>` (`bvector_cos_ops`): cosine distance, defined as $1 - \frac{\Sigma x_iy_i}{\sqrt{\Sigma x_i^2 \Sigma y_i^2}}$.
```
Comment (silver-ymz) [inline src/usage/vector-type.md:24]: We also support `jaccard` distance `<~>`, which defined as $1 - \frac{|X\cap Y|}{|X\cup Y|}$. And we can mention that `hamming` distance is the same with squared Euclidean distance.

**Comment by gaocegege** [comment]
Comments are addressed, please take a look.
Thanks for your time :smile:

**Review by usamoi — APPROVED** [review]

**Pull request merged**